### PR TITLE
Feature/querystring/date

### PR DESCRIPTION
**Pull request by Chabelle78**
Query string

### CODE REVIEW

File: src/api/projects/controllers/createProjectUser.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  try {
+    await prisma.project.update({
+      where: {
+        id: projectId,
+      },
+      data: {
+        users: { connect: { id: userId } },
+      },
+    });
+    res.sendStatus(201).json({ message: "Ok done for me projects" });
```
Comment (MatthiasWanner) [inline src/api/projects/controllers/createProjectUser.js:14]: You can't write anything... 😂. 
Simple that explain what was the action. Like "User added to project"

File: src/api/users/controllers/createUserProject.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  try {
+    await prisma.user.update({
+      where: {
+        id: userId,
+      },
+      data: {
+        projects: { connect: { id: projectId } },
+      },
+    });
+    res.sendStatus(201).json({ message: "done for me" });
```
Comment (MatthiasWanner) [inline src/api/users/controllers/createUserProject.js:14]: Same as for the other route: explicit message please.

**Review by MatthiasWanner — CHANGES_REQUESTED** [review]

File: src/api/companies/controllers/getProjectsFromCompany.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  try {
+    const allProjects = await prisma.company.findUnique({
+      where: {
+        id,
+      },
+      include: {
+        projects: true,
+      },
+    });
+    res.status(200).json(allProjects.projects);
```
[the remark this answers is not in the document]
Comment (Chabelle78) [reply src/api/companies/controllers/getProjectsFromCompany.js]: Ok done

File: src/api/projects/controllers/createProjectUser.js (hunk 1 above)
Comment (Chabelle78) [reply src/api/projects/controllers/createProjectUser.js]: Ok done / only sendstatus

File: src/api/users/controllers/createUserProject.js (hunk 2 above)
Comment (Chabelle78) [reply src/api/users/controllers/createUserProject.js]: ok done / in previous pr

**Review by Aeryle — APPROVED** [review]

**Review by ThomasBarrial — APPROVED** [review]
cmldt

**Review by MatthiasWanner — APPROVED** [review]

**Pull request merged**